CONTRIBUTING.md

```diff
@@ -1,3 +1,12 @@
+# Contribution Guidelines
+
+The following documents constitutes a set of guidelines to which contributors must adhere.
+
+* [Contributions Licensing](#contributions-licensing)
```

Comment:
Should we add code style to this list?

Reply:
I agree, although I'd do it in a separate PR